music21/musicxml/test_xmlToM21.py

```diff
@@ -619,6 +619,18 @@ def testOrnamentC(self):
                     count += 1
         self.assertEqual(count, 1)
 
+        uppercount = 0
+        lowercount = 0
```

Comment:
camelCase or underscore_case.  (for accessibility)

Reply:
Done.